### ISSUE TITLE
Assess distribution of congenital phenotypes

### ISSUE THREAD

**Issue by bschilder** [neurogenomics/rare_disease_celltyping#68]
@KittyMurphy 

@NathanSkene and I discussed whether congenital phenotypes should be excluded from the gene therapy target prioritisation pipeline. 
@NathanSkene suggested that congenital phenotypes should be removed because they are mostly associated with morphological/gross anatomical abnormalities which would not be amenable to gene therapy. 

I agree that morphological phenotypes wouldn't be good candidates for gene therapy, but I think removing all congenital phenotypes isn't the best way to filter out these scenarios. Especially when we have "physical_malformations" as a GPT annotation we can use to filter on directly.

Some thing we need to check:
- [x] What proportion of HPO phenotypes are congenital?
- [x] What proportion of our phenotype-cell associations would be left if we filtered on congenital phenotypes?
- [x] Which branches of the HPO are most enriched for congenital phenotypes? (figure for GPT HPO paper)
- [x] Which branches of the HPO are most enriched for phenotypes with fetal cell type associations? (figure for Rare Diseases paper)
- [x] Are congenital phenotypes significantly enriched for physical malformation phenotypes? 
- [x] What proportion of phenotypes and/or diseases with existing gene therapies are congenital (eg. SMA, sickle-cell anaemia, haemophilia, thalassemia, Leber congenital amaurosis)?

**Comment by bschilder** [comment]
# What proportion of HPO phenotypes are congenital?

```R
gpt_check <- HPOExplorer::gpt_annot_check()
table(gpt_annot$annot$congenital_onset)
```
![Image](https://github.com/neurogenomics/rare_disease_celltyping/assets/34280215/d6d6ef48-253c-4355-a75f-f203bb496f1a)

There's a fairly even distribution across response categories.

# What proportion of our phenotype-cell associations would be left if we filtered on congenital phenotypes? 

```R
results <- MSTExplorer::load_example_results()
results <- results[q<0.05]
results <- HPOExplorer::add_gpt_annotations(results)
table(results$congenital_onset, useNA = "always")/nrow(results)*100
```


![Image](https://github.com/neurogenomics/rare_disease_celltyping/assets/34280215/75246ecc-58b0-435e-b81b-0caa012a4725)

- 17% of associations are with phenotypes that are never congenital (0)
- 51.5%  of associations are with phenotypes that are never or rarely congenital (0-1)


# Which branches of the HPO are most enriched for congenital phenotypes? (figure for GPT HPO paper)

Ordered by percentage of phenotypes that are always congenital. Note that "abnormality of prenatal development or birth" is no.1. Neoplasms and immune disorders are further down.

Will add this to the GPT paper.

## All HPO terms

![Image](https://github.com/neurogenomics/rare_disease_celltyping/assets/34280215/18595695-37f8-46fe-ae88-3842322fbb66)

## Only HPO terms under "Phenotypic abnormality"

![Image](https://github.com/neurogenomics/rare_disease_celltyping/assets/34280215/7138b6bf-995c-410e-9225-b0cf179d96f7)

**Comment by bschilder** [comment]
# Which branches of the HPO are most enriched for phenotypes with fetal cell type associations? (figure for Rare Diseases paper)

Abnormality of limbs has is the most enriched for fetal celltypes.

![Image](https://github.com/neurogenomics/rare_disease_celltyping/assets/34280215/691c2038-f52a-46dc-ac0a-b76522a323e6)

**Comment by bschilder** [comment]
#  Are congenital phenotypes significantly enriched for physical malformation phenotypes?

Yes, there is a 68% correlation between these two metrics using the most stringent approach (always, often, rarely, never all treated as different values, encoded numerically from 3-0).

![Image](https://github.com/neurogenomics/rare_disease_celltyping/assets/34280215/3aec07dc-14a0-4a99-84ed-49ee14bdfebc)

**Comment by bschilder** [comment]
# What proportion of phenotypes and/or diseases with existing gene therapies are congenital (eg. SMA, sickle-cell anaemia, haemophilia, thalassemia, Leber congenital amaurosis)?

Ultimately, what we're really asking is whether gene therapy (or any kind of therapy) will be effective for treating a phenotype once the phenotype has already manifested.  There are several implicit assumptions here:
1. **Treatments cannot be applied prenatally**
2. **Treatments are ineffective once phenotypes have manifested**

## 1. Treatments cannot be applied prenatally
This has historically been true but there are now techniques being more commonly used in hospitals such as [prenatal surgery](https://my.clevelandclinic.org/health/treatments/24945-fetal-surgery). 

There have also been recent successes with the **prenatal** treatment of genetic disorders like [Pompe disease using enzyme replacement therapy](https://www.ucsf.edu/news/2022/11/424166/fetal-therapy-offers-hope-patients-rare-genetic-disorders). 
> "This is an important step toward more curative therapies that could be given prenatally – like gene therapy and gene editing."

So while it's still early days for prenatal treatment with biologics, I expect we'll be seeing more of this in the near future.  

## 2. Treatments are ineffective once phenotypes have manifested

See below for some cases where this is (currently) true and several where it is not true.

## Cases where therapy post-manifestation is ineffective
### SMA
@NathanSkene and I discussed this and concluded that there is a distinction to be made between diseases being congenital and phenotypes being congenital. In fact, just today I listened to a podcast where Wendy Chung mentioned that phenotypes of SMA often don't appear until 6mo after birth, at which point it can be too late as the child will likely die by 12mo of age. 
https://youtu.be/4MUwzYre5PU?si=-i8UtEdTvuT5ADCb&t=1661

### Physical malformations 
Many physical malformations due to developmental disorders are not currently treatable with gene therapy. Here, surgery + other medications may be more effective (eg cleft palate, hydrocephalus, agenesis of the corpus collosum). Gene therapy could still be helpful for these patients for preventing the physical malformation from re-emerging, such as in my example of skeletal dysplasia where initial surgery on the rib cage could allow the child to breath while follow up with gene therapy might allow the chondrocytes to grow normally and thus not cause constriction of the organs as the child continues to grow.

## Cases where therapy post-manifestation is effective
### Blood disorders: SCA, thalassaemia
On the other hand, diseases like Sickle Cell Anemia are congenital. Some phenotypes of SCA are also congenital, such as ['Sickled erythrocytes'](https://hpo.jax.org/app/browse/term/HP:0030058), while [others appear several months after birth](https://www.nhs.uk/conditions/sickle-cell-disease/symptoms/#:~:text=Sickle%20cell%20disease%20can%20cause,painful%20episodes). Same goes for thalassaemia. Despite the fact that some of the core phenotypes are congenital in these diseases, these are some of the most successful gene therapies to date.

## DMD
Duchenne Muscular Dystrophy (DMD) first begins to manifest as phenotypes around 3-4 years of age. Existing gene therapies don't restore function that's already been lost at the time treatment begins, but it can slow (and possibly stop) the progression of the primary phenotypes such as muscle weakness. As with many diseases, earlier detection and diagnosis is better but gene therapy still can have profound effects even post-phenotype manifestation.
https://www.chop.edu/gene-therapy-duchenne-muscular-dystrophy

# Conclusions

1. Treating physical malformation phenotypes (post-manifestation) using gene therapy is probably far off. However, gene therapy can still be helpful for preventing these malformations from reemerging after surgical interventions. 
2. Treating non-physical malformation phenotypes post-manifestation can effective for some phenotypes. For blood disorders like SCA and thalassaemia, the gene therapy can be applied many years post-manifestation to great effect (basically curing individuals). This is partly due to the fact that there is not a lot of permanent tissue damage done by the disease. DMD can also be treated post-manifestation, but earlier treatment is generally associated with better outcomes as this disorder does cause permanent damage to some tissues. On the other end of the spectrum, you have SMA where kids often don't manifest phenotypes [until 6mo, at which point it's too late to save them with gene therapy which is currently only effective if administered <3mo of age](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10104684/#:~:text=There%20is%20substantial%20evidence%20that,under%20three%20months%20of%20age.). This is why genetic testing (of parents and the child) is critical for treating SMA early. 

So basically, whether a phenotype can be treated post-manifestation (whether it's congenital or non-congenital) really depends on the particular phenotype. After excluding physical malformations, I don't see any strong evidence that congenital vs. non-congenital is a good predictor of treatability by gene therapy. It all depends on 1) the rate of progression (eg DMD is relatively slow, SMA is fast), 2) whether the particular phenotype causes permanent tissue damage.

**Comment by bschilder** [comment]
It's also worth mentioning, that there's plenty of non-congenital phenotypes that aren't treatable by gene therapy. For example, heart attack.
https://hpo.jax.org/app/browse/term/HP:0001658

**Comment by bschilder** [comment]
There are some scenarios where non-congenital phenotypes are more amenable to [gene] therapy than congenital phenotypes:
- prenatal therapy is not available
- the risk for the phenotype can be accurately predicted *before* its manifestation. you can do this with genetics in cases where the parents happened to do genetic screening, and the genetic variant is highly penetrant (not always the case even with rare mutations). however, in most cases genetic screening only occurs **after** the child starts manifesting phenotypes. so non-congenital doesn't necessarily mean opportunity for early detection.

**Comment by NathanSkene** [comment]
I’m leaning towards that your idea of filtering by malformations, rather than congenital onset might be the right one.  Above results sound great from a skim read, looking forward to discussing.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
